draft-packages/visually-hidden/KaizenDraft/VisuallyHidden/styles.module.scss

```diff
@@ -0,0 +1,9 @@
+@import "~@kaizen/component-library/styles/utils";
+
+// If the .visually-hidden class is applied to natively focusable elements
+// (such as a, button, input, etc) they must become visible when they receive
+// keyboard focus. Otherwise, a sighted keyboard user would have to try and
+// figure out where their visible focus indicator had gone to.
+.visuallyHidden:not(:focus):not(:active) {
+  @include sr-only();
```

Comment:
It'd be great to move to a world where components are a little more independent of one another. I hope that with a component available to visually hide content we will no longer require the sass mixin from `component-library`. 

Could we duplicate the mixin in this component and drop the `@kaizen/component-library` (currently missing) in `VisuallyHidden`'s package?